site/_docs/history.md

```diff
@@ -56,20 +132,221 @@ other software versions as specified in gradle.properties.
 #### New features
 {: #new-features-1-33-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-2884">CALCITE-2884</a>]
+  Implement JSON_INSERT, JSON_REPLACE, JSON_SET
```

Comment:
These JSON functions are considered as new features while others such as `ST_CoveredBy` as minor enhancements. Consider moving them around for consistency.

Reply:
moved this one to "New Features"

Reply:
+1 All new SQL functions should be regarded as new features. The GIS aggregregate functions are pretty significant.

I think every function added should appear explicitly named in the release notes. This even applies to CALCITE-5180 and related changes, which add a lot of functions to the BigQuery library. This may mean changing the commit messages as they appear in the release notes but that's OK.

Reply:
moved new functions to "new features" section and GIS aggregates entry copied to notable section